src/libcore/ops/range.rs

```diff
@@ -904,7 +904,7 @@ impl<T> RangeBounds<T> for RangeTo<&T> {
 }
 
 #[stable(feature = "collections_range", since = "1.28.0")]
-impl<T> RangeBounds<T> for Range<&T> {
+impl<T: ?Sized> RangeBounds<T> for Range<&T> {
     fn start_bound(&self) -> Bound<&T> {
         Included(self.start)
     }
```

Comment:
Was this change supposed to apply to the RangeBounds\<T\> impls for RangeInclusive\<&T\> and RangeToInclusive\<&T\> (below) as well?